examples/lighting-app/telink/CMakeLists.txt

```diff
@@ -96,8 +96,6 @@ target_sources(app PRIVATE
                ${CHIP_ROOT}/src/app/clusters/color-control-server/color-control-server.cpp
                ${CHIP_ROOT}/src/app/clusters/occupancy-sensor-server/occupancy-sensor-server.cpp
                ${CHIP_ROOT}/src/app/clusters/network-commissioning/network-commissioning.cpp
-               ${CHIP_ROOT}/src/app/clusters/network-commissioning-old/network-commissioning-ember.cpp
```

Comment:
A bit confused for the change to telink build file since other files are all under esp32 domain.